### ISSUE TITLE
The Missing Configs

### ISSUE THREAD

**Issue by Konix26** [GriefPrevention/GriefPrevention#520]
**Hello, I need suggestions in how to fix this issue, my goal is to suggest or inform of a way to fix the issue to the owner of the server I play on, and remove the limit to claims that is now locked on the server. I can't seem to find any related issues on here, or throughout google and don't know how to recreate the problem myself, therefor I have resorted to asking professionals. :p**

The Problem: I'm not the owner of this server, nor an administrator, however I have a fair bit of information and would like a suggestion in how to approach this issue, including reporting suggestions in how to fix the issue. This server I play on has an issue where claim limits are set for players, this was not intended, and as I read in many ways that the plugin is set, the default is no limit. I've asked about this and they said it's meant to be unlimited, however they cannot find the config to fix this, or there is no config available to fix it. I don't know much about applying plugins or running servers, so I'm kinda at a loss here myself, but I'd like to help as much as I can. The issue I'm personally having is I have a large claim set, and I have tried to subdivide claims for it, however I cannot. When I use an alternate account to do so, I can subdivide claims that way, but the limit to it seems inconsistent, and when I attempt to lay any other claims outside of subdivisions it disallows this action, I also cannot subdivide claims but can remove actual claims and replace actual claims without being able to subdivide more. So really I don't know if this is a bug, or just a question but for now simply assume it as a question.

TLDR; _I cannot add subdivisions despite removing normal claims, I can use an alternate account to lay subdivisions but that account has reached a supposed limit on subdivisions and cannot lay normal claims now, these limits aren't meant to exist and the config seems to be missing._

EDIT: _I'll answer any questions to the best of my ability, going as far as asking the owner of the server if needed._ **Thanks in advanced for any help or suggestions that may be provided.**

**Comment by TomLewis** [comment]
Yes there is, plugins/GriefPreventionData/config.yml and look for Max Accrued Claim Blocks

**Comment by Konix26** [comment]
> Yes there is, plugins/GriefPreventionData/config.yml and look for Max Accrued Claim Blocks

That's the problem, they cannot find this file or It is not in the location it is said to be in.

**Comment by bigpresh** [comment]
The config file is, as stated, `plugins/GriefPreventionData/config.yml` (from the root of the server directory).

If the administrator of your server can't find the file, I'd suggest to them to look for it from the root of the server's directory, or use `find` or similar to locate it.

For example, they could try to locate the folder the config file will be in using: `find / -type d -name GriefPreventionData`.

Either way, I'm not sure there's much we can do to help you, the admin of the server will need to take care of it.

**Comment by bigpresh** [comment]
(You've also deleted the whole issue template, and omitted information like what you are trying to do, what you expect to happen, what error messages you get, any log messages, what GP version and Spigot version you're using, etc, which just makes it even harder to be of any use.)

**Comment by RoboMWM** [comment]
Yea, for all we know it may be a sponge server.

Max claim limit is disabled by default, so _somebody_ knows where it is.

**Comment by Konix26** [comment]
> (You've also deleted the whole issue template, and omitted information like what you are trying to do, what you expect to happen, what error messages you get, any log messages, what GP version and Spigot version you're using, etc, which just makes it even harder to be of any use.)

Unfortunately the server runs Sponge API, not Bukkit/Spiggot.

**Comment by RoboMWM** [comment]
Ok, then it is not running this plugin.

**Comment by bigpresh** [comment]
@Konix26 for clarity, there's two versions of GriefPrevention out there - this one, which is the original Spigot (and Paper) version, and a separate version for Sponge-powered servers, managed by entirely different people - that version can be found at https://ore.spongepowered.org/blood/GriefPrevention

I would recommend you point your server's owner at it and have them interact with the developers of that version if they need help, though, rather than acting as middle-man, because that just leads to confusion and frustration.

**Comment by Konix26** [comment]
> @Konix26 for clarity, there's two versions of GriefPrevention out there - this one, which is the original Spigot (and Paper) version, and a separate version for Sponge-powered servers, managed by entirely different people - that version can be found at https://ore.spongepowered.org/blood/GriefPrevention
> 
> I would recommend you point your server's owner at it and have them interact with the developers of that version if they need help, though, rather than acting as middle-man, because that just leads to confusion and frustration.

I'm not acting as the middle man, I'm acting on my own because I don't want to bother the owner in the first place, if I find the answer first and then just give them the information it'll be easier for everyone.

Thanks for the link

**Comment by RoboMWM** [comment]
This is off topic for an issue tracker, but

> I'm not acting as the middle man

> if I find the answer first and then just give them the information

These statements are contradictory. You don't have access to information that may be requested by whoever you're asking for help from, so you are a middleman, albeit self-initiated. (You can, however, take a look at the documentation yourself though and try to figure out what they need to do for example.)
